eval/eval_openlm_ckpt.py

```diff
@@ -104,11 +112,14 @@ def main():
 
     print("Loading model into the right classes")
     open_lm = OpenLMforCausalLM(OpenLMConfig(create_params(args)))
-    tokenizer = GPTNeoXTokenizerFast.from_pretrained("EleutherAI/gpt-neox-20b")
+    if "gpt-neox-20b" in args.tokenizer:
+        tokenizer = GPTNeoXTokenizerFast.from_pretrained("EleutherAI/gpt-neox-20b")
+    elif "llama" in args.tokenizer:
+        tokenizer = LlamaTokenizerFast.from_pretrained(args.tokenizer)
 
     state_dict = checkpoint["state_dict"]
     state_dict = {x.replace("module.", ""): y for x, y in state_dict.items()}
-    open_lm.model.load_state_dict(state_dict)
+    open_lm.model.load_state_dict(state_dict, strict=False)
```

Comment:
is this because llama has extra keys?

Reply:
Actually the second point should not be a problem as long as we load with the same positional embedding it might be a bug I'll check that

Reply:
There was a bug in the create_params function that I miss-interpreted as a difference in state_dicts. This is solved.